crates/iota-kvstore/src/main.rs

```diff
@@ -19,25 +18,17 @@ use iota_types::{base_types::ObjectID, digests::TransactionDigest, storage::Obje
 use prometheus::Registry;
 use telemetry_subscribers::TelemetryConfig;
 
-#[derive(Debug, Clone, Copy, Default, ValueEnum)]
+#[derive(Debug, Clone, Copy, Default, ValueEnum, strum::Display)]
```

Comment:
I think you can do this:
```suggestion
#[derive(Debug, Clone, Copy, Default, ValueEnum, strum::Display)]
#[strum(serialize_all = "snake-case"]
```

Reply:
@DaughterOfMars fixed: https://github.com/iotaledger/iota/pull/7637/commits/3e4e090b3db448ff5f67d6e44415d4be99c122c2